roles/confluent.variables/defaults/main.yml

```diff
@@ -151,6 +151,9 @@ zookeeper_client_port: "{{'2182' if zookeeper_ssl_enabled|bool else '2181'}}"
 ### SASL Mechanism for Zookeeper Server to Server and Server to Client Authentication. Options are none, kerberos, digest. Server to server auth only working for digest-md5
 zookeeper_sasl_protocol: "{{sasl_protocol if sasl_protocol == 'kerberos' else 'none'}}"
 
+### Only use to customize directory Zookeeper writes log files to. Default location is /var/log/kafka. NOTE- zookeeper.log_path is deprecated.
```

Comment:
Modify this variable to customize the directory that Zookeeper writes log files to. Default location is /var/log/kafka. NOTE- zookeeper.log_path is deprecated.